app/helpers/blacklight/catalog_helper_behavior.rb

```diff
@@ -0,0 +1,276 @@
+# -*- encoding : utf-8 -*-
```

Comment:
What error does it lead to? I've just tried removing it and functionality seems to work as expected for me. Am I missing something?